shacl12-core/index.html

```diff
@@ -2064,7 +2064,9 @@ <h3>Property Shapes</h3>
 					that is the <a>subject</a> of a <a>triple</a> that has <code>sh:path</code> as its <a>predicate</a>.</span>
 					<span data-syntax-rule="path-maxCount">A shape has at most one <a>value</a> for <code>sh:path</code>.</span>
 					<span data-syntax-rule="path-node">The <a>value</a> of <code>sh:path</code> in a property shape is a <a>well-formed</a>
-					<a>SHACL property path</a>.</span>
+					<a>SHACL property path</a>. If a property shape contains a <code>sh:ReifiableByConstraintComponent</code>,
+					<code>sh:path</code> values are constrained to IRIs.</span>
```

Comment:
I would move this into the sh:reifiableBy definition itself, e.g. "The $path is an IRI" or "The sh:path MUST be an IRI". Whenever any constraint definition is ill-formed, the whole shapes graph is invalid and thus can produce a failure. I believe it's better to bundle this rule where it belongs because there are already other exceptions on sh:path, e.g. sh:values can only be used for IRIs, and moving it into sh:reifiableBy keeps the document more modular.